content/api/plugins/configuration-api.md

```diff
@@ -51,7 +51,7 @@ modified from the `setupNodeEvents` function in the Cypress configuration.**
 
 </Alert>
 
-Resolved values will show up in the "Settings" tab of the Cypress App.
+Resolved values will show up in the "Settings" tab of the Cypress Test Runner.
```

Comment:
I think we can omit Test Runner here because the "Settings" page is a feature of the app, not just Test Runner.